views/cart.hbs

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Shopping App</title>
+    {{!--
+    <meta name="viewport" content="width=device-width" /> --}}
+    <link rel="stylesheet" href="../static/styles/cart.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Dosis:400,600,700">
+</head>
+
+<body>
+    <div>
+        <p>
+            My Cart({{0}})
+        </p>
+        <button>X</button>
+    </div>
+    <div class="cartDetails">
+        <section >
+            <img src='../static/images/products/fruit-n-veg/apple.jpg' style="height: 100%;object-fit:cover">
+        </section>
+        <section class="itemdetails">
+            <h3>Apple- washington Regular - 4pcs</h3>
+            <section>
+            <button class="round">+</button>
+            <span>1</span>
+            <button class="round">-</button>
+            <span>X</span>
+            <span>Rs.187</span>
+            <span class="total">Total</span>
+        </section>
+        </section>
+    </div>
+    <div class="promotionLogo">
+        <img src="../static/images/lowest-price.png">
+        <p>You won't find it cheaper anywhere</p>
+    </div>
+    <footer>
+        <span>
+            Promo code can be applied on payment page
+        </span>
+        <button>Proceed to Checkout </button>
+    </footer>
```

Comment:
Handlebars allows you to create a common layout or a template file in which you can insert partial html like footer header etc. This way you don't have to add footer header on every page